VirtoCommerce.CatalogModule.Data/Search/ProductSearchRequestBuilder.cs

```diff
@@ -153,6 +153,11 @@ protected virtual IList<IFilter> GetPermanentFilters(ProductSearchCriteria crite
                 result.Add(FiltersHelper.CreateTermFilter("__type", criteria.ClassTypes));
             }
 
+            if (!string.IsNullOrEmpty(criteria.ProductType))
```

Comment:
You can use this property for this purpose https://github.com/VirtoCommerce/vc-module-core/blob/master/VirtoCommerce.Domain/Catalog/Model/Search/ProductSearchCriteria.cs#L24